app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class CommentsController < ApplicationController
```

Comment:
```suggestion
class CommentsController < ApplicationController
  before_action :set_commentable
```

親クラス側で set_commentable を呼ぶようにするとDRYになってよいかも